AlphaWallet/Tokens/ViewControllers/TokenInstanceViewController.swift

```diff
@@ -121,21 +121,22 @@ class TokenInstanceViewController: UIViewController, TokenVerifiableStatusViewCo
 
         header.configure(viewModel: .init(tokenObject: tokenObject, server: tokenObject.server, assetDefinitionStore: assetDefinitionStore))
 
-        let actions = viewModel.actions
-        buttonsBar.numberOfButtons = actions.count
-        buttonsBar.configure()
-        for (action, button) in zip(actions, buttonsBar.buttons) {
+        buttonsBar.configure(.combined(buttons: viewModel.actions.count))
+        buttonsBar.viewController = self
+
+        for (index, button) in buttonsBar.buttons.enumerated() {
+            let action = viewModel.actions[index]
             button.setTitle(action.name, for: .normal)
             button.addTarget(self, action: #selector(actionButtonTapped), for: .touchUpInside)
             switch account.type {
             case .real:
                 if let selection = action.activeExcludingSelection(selectedTokenHolders: [tokenHolder], forWalletAddress: account.address) {
                     if selection.denial == nil {
                         button.isHidden = true
-                    } else {
+                    } else if button.isHidden != true {
```

Comment:
This else-if seems redundant:

```
                    } else if button.isHidden != true {
                         button.isHidden = false
```

Is it needed?

Reply:
@vladyslav-iosdev this doesn't seem to be fix yet?

Reply:
i removed it, but seems like during rebase it returned back. i will fix it